.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macos-10.15, windows-2019]
-        sofa_branch: [master]
+        sofa_branch: [master, v21.12]
```

Comment:
I've already added this in the branch v21.12 and I don't think we should have that on master right?
The goal is not to keep master working with this version.

Reply:
It's actually a good idea to keep it to know when master will not be compatible with v21.12 anymore (and to remove it at this moment).